UIKit/_UIBackdropContentView.h

```diff
@@ -0,0 +1,6 @@
+@interface _UIBackdropContentView : UIView
+
+- (void)recursivelyRemoveBackdropMaskViewsForView:(id)arg1;
+- (void)backdropView:(id)arg1 recursivelyUpdateMaskViewsForView:(id)arg2;
```

Comment:
Can these types be precise?
`backdropView` instead of `arg1`
`view` instead of `arg2`